tests/config/config.conf

```diff
@@ -28,6 +28,8 @@ errorVariable = true
 
 # hyprlang noerror false
 
+categoryKeyword = oops, this one shouldn't call the handler, not fun
```

Comment:
if it's a category one, this should throw an error, no?

Reply:
Well, in my eyes `categoryKeyword` and `category::categoryKeyword` are effectively totally different keywords. The naming here might feel a little bit confusing but it's to ensure the handler for the latter is not called for the former as well. This one is intended to be a legit config option but that shouldn't be picked up by the handler.

Reply:
oh, right. Missed that, because both variables are missing a test case :P